.github/CODEOWNERS

```diff
@@ -8,19 +8,80 @@
 # The go-samples-reviewers team is the default owner for anything not
 # explicitly taken by someone else.
 *                                     @GoogleCloudPlatform/go-samples-reviewers
+/docs/                                @GoogleCloudPlatform/go-samples-reviewers
+/getting-started/                     @GoogleCloudPlatform/go-samples-reviewers
```

Comment:
it feels like getting-started is owned by all DEE as well.

Reply:
Good point. It doesn't look like there's a `dee-all` team in GoogleCloudPlatform. There is a `cloud-dpe` though--should I set that as CODEOWNER for /getting-started/ ?